EIPS/eip-3369.md

```diff
@@ -0,0 +1,183 @@
+---
+eip: 3369
```

Comment:
Any certain reason for thing being EIP-3372 instead of EIP-3369 (except the pull request name)?

There were a couple highly-upvoted reddit posts that called it an EIP-3369. Also, 3369 has the same two numbers in the end as EIP-969 does, which looks pretty nice.

Reply:
EIP numbers are assigned by editors.  We have a system in place for numbering them and it should be considered arbitrary to users.  We definitely do not support people picking their own numbers, squatting on numbers, or manipulating the number selection system to choose particular values.  Think of it like an auto-generated number in a database... you get what you get.